docs/_layouts/area-page.njk

```diff
@@ -4,19 +4,6 @@ layout: default.njk
 
 {% include "page-title.njk" %}
 <div class="p-lg-16">
-  <div class="container-fluid">
-    <div class="row">
-      <div class="col-12 content">
-        {{ content | safe }}
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-12">
-        <hr class="minor mt-1 mb-6">
-      </div>
-    </div>
-  </div>
```

Comment:
If the area pages are not going to provide any text info above the cards, could we remove that info from the md files or is that going to be used elsewhere?

Reply:
that could be used elsewhere so we’re leaving it in the MD for now @frogpad7

Reply:
Sounds good; everything else works as I would expect it to.